metrics/user_registries.go

```diff
@@ -0,0 +1,811 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
dskit is Apache 2. I'm also not sure if we should be using these `Provenance` headers since we don't elsewhere.